src/main/kotlin/com/kafkakotlin/demo/users/UserService.kt

```diff
@@ -0,0 +1,21 @@
+package com.kafkakotlin.demo.users
+
+import org.springframework.stereotype.Service
+
+@Service
+class UserService(
+    val userRepository: UserRepository
+) {
+
+    fun createUser(user: User) {
+        return userRepository.createUser(user)
+    }
+
+    fun getUsers(): Map<String, User> {
+        return userRepository.getUsers()
+    }
+
+    fun getRemoteUsers(): Map<String, User> {
```

Comment:
a method called `getRemoteUsers()` calls `getLocalUsers()`.... If I saw that I'd think it was a bug... are both named right?

Reply:
Yeh I see what you mean. My naming demons strike again! I'll have a think...